### DISCUSSION TITLE
CopyQ latest version not supported on Apple M3 chip?

### DISCUSSION THREAD

**Discussion by PratapJi** [hluk/CopyQ#2827] [General]
Tried to install copyQ [CopyQ-macos-12-m1.dmg.zip](https://github.com/hluk/CopyQ/releases/download/v9.0.0/CopyQ-macos-12-m1.dmg.zip) to Mac M3 but getting "CopyQ is damaged and can't be opened"

<img width="319" alt="image" src="https://github.com/user-attachments/assets/e014f38d-29a2-4a13-a93f-42a7dc601a1c">

**Comment by Brunni132** [comment]
Known issue: https://github.com/hluk/CopyQ/issues/2652#issuecomment-2018283626

**Comment by Brunni132** [comment]
Also, remember that, after an update, you'll have to replace the entry in the system accessibility option (system settings -> privacy & security -> Accessibility -> select CopyQ, click on the minus button down-left, enter the password, then click on the plus sign, and add copyq).
